test/travis_test/build_2_container.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+BUILD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+code_dir=$(dirname $BUILD)
+code_dir=$(dirname $code_dir)
+echo "build path is $BUILD"
+echo "code path is $code_dir"
+
+# Initialize the needed mizar submodule
+git submodule update --init --recursive
+
+# Create and Start the build contrainer
+docker build -f $code_dir/build/Dockerfile -t aca_build0 .
+docker rm -f aca_PARENT || true
+docker create -v $code_dir:/mnt/host/code -it --privileged --cap-add=NET_ADMIN --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --name aca_PARENT aca_build0:latest /bin/bash
+docker start aca_PARENT
+docker rm -f aca_CHILD || true
+docker create -v $code_dir:/mnt/host/code -it --privileged --cap-add=NET_ADMIN --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --name aca_CHILD aca_build0:latest /bin/bash
+docker start aca_CHILD
+
+# Build mizar first
+# docker exec a1 bash -c "cd /mnt/host/code/mizar && make"
+
+if [ "$1" != "test" ]; then
```

Comment:
this if condition will be executed for our case: ./test/travis_test/build_2_container.sh 2_port_test
my suggestion is to remove this if condition by removing line 24 to 28, and then change line 29 to if [ "$1" == "2_port_test" ]; then